src/store/albums.js

```diff
@@ -12,6 +12,7 @@ import {
   compareAlbumsByReleaseFirst,
   compareAlbumsByTitleFirst,
 } from "../comparators";
+import store from "./store";
```

Comment:
Importing the store from a store submodule doesn't feel very nice. Is there no way around this?

Reply:
Not that I'm aware of. I made some attempts to have the `restored` property as part of the module, or kept inside the store itself, but didn't find a way to make it work

Reply:
OK :(